docs/sdks/uid2-sdk-ref-android.md

```diff
@@ -110,19 +113,62 @@ To install with Maven, add the SDK as a dependency in the `pom.xml` file:
 <dependency> 
   <groupId>com.uid2</groupId> 
   <artifactId>uid2-android-sdk</artifactId> 
-  <version>0.5.0</version> 
+  <version>1.2.0</version> 
 </dependency> 
 ```
 
 ## Usage Guidelines
 
-The initial UID2 Identity *must* be generated by the implementing application and then passed into the UID2 SDK. Because of security requirements, the UID2 Mobile SDKs cannot create new UID2 identities.
+The **UID2Manager** singleton is the primary developer API for the UID2 SDK for Android. It is responsible for storing, refreshing, and retrieving UID2 Identity and according token.
+
+You must initialize the `UID2Manager` manually before you can use it, as the SDK expects the singleton to be initialized before use. This does two things:
+
+-   It allows for easier access later.
+-   It allows the consuming application to potentially provide its own network instance, responsible for making requests.
+
+The initialization can be done during the creation of the APPLICATION instance, as shown in the following example:
+
+```js
+class MyApplication : Application() {
+  override fun onCreate() {
+    super.onCreate()
+   // Initialize the UID2Manager class. Use DefaultNetworkSession rather than providing our own
+   // custom implementation. This can be done to allow wrapping something like OkHttp.
+   UID2Manager.init(this.applicationContext)
+```
+
+There are two ways to establish an initial UID2 Identity:
+
+1. Generate the UID2 identity using DII - email (hash) or phone no (hash). For integration instructions, see [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md).
+
+2. Create a UID2 identity server-side and then pass it into the UID2 SDK. For integration instructions, refer to [Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md).
 
 The UID2 Mobile SDKs can perform refreshes of UID2 identities, after an Identity is established. This is because the refresh functionality relies on the refresh tokens that are part of the UID2 Identity.
 
-The **UID2Manager** singleton is the primary developer API for the UID2 Android and iOS SDKs. It is responsible for storing, refreshing, and retrieving UID2 Identity.
+## Code Samples
+
+The following code samples provide examples of performing specific activities relating to managing UID2 with the UID2 SDK for Android.
+
+Generate an initial UID2 Identity (refer to [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side#configure-the-uid2-mobile-sdk)):
+``` javascript
```

Comment:
I think all the Android snippets are in Kotlin.

Reply:
Yep. I emulated the tags I'd used in the Android and iOS SDK docs... I consulted wit Brad at the time. Happy to use something different but Sunny says it looks the same... possible that we need a Docu config change on this one. hopefully we can follow up with further research on that?

Reply:
yes let's follow up on a separate ticket if you could create 1 @genwhittTTD please

Reply:
There's a minor difference in syntax highlighting, but not worth fixing now.

Swift and Kotlin are supported by default https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages